tap_wikipedia/tap.py

```diff
@@ -21,33 +24,31 @@ class Tapwikipedia(Tap):
             "settings",
             th.ObjectType(
```

Comment:
Nit: my understanding is that Singer's typing is just shorthand for constructing JSON schemas. You should be able to get the same schema out of Pydantic:
```
https://docs.pydantic.dev/latest/concepts/json_schema/
```
I'd probably eyeball it to ensure equivalence first.

Reply:
Yes, Pydantic's schema can be used. I have added it.